receiver/prometheusremotewritereceiver/.chloggen/receiver_prometheusremotewritereceiver_countermetrics.yaml

```diff
@@ -0,0 +1,4 @@
+# Add proper counter metric handling
```

Comment:
This is still not the right place. The .chloggen directory is the one at the root directory: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/.chloggen